### PR TITLE
Studio: Add context to api

### PR DESCRIPTION
**Pull request by kozer**
## Proposed Changes

TBA

## Testing Instructions
TBA

## Pre-merge Checklist

- [ ] Have you checked for TypeScript, React or other console errors?

### CODE REVIEW

File: src/hooks/use-assistant-api.ts [hunk 1]
```diff
@@ -7,13 +7,20 @@ export function useAssistantApi() {
...
 			setIsLoading( true );
 			const body = {
 				messages,
+				chat_id: chatId ?? undefined,
+				context: {
+					plugins: [ 'jetpack' ],
+					wp_version: '5.8.1',
+					php_version: '7.4.24',
+					number_of_sites: 4,
+				},
```
Comment (sejas) [inline src/hooks/use-assistant-api.ts:23]: Let's add the real data here.

**Review by sejas — COMMENTED** [review]
@kozer , this PR shows how easy is to migrate to the Odie backend.
If you prefer we can delegate the forntend to another dev. Let me know what you think.

File: src/hooks/use-assistant.ts [hunk 2]
```diff
@@ -7,28 +7,52 @@ export interface Message {
...
 
 	const clearMessages = () => {
 		setMessages( [] );
-		localStorage.setItem( selectedSiteId, JSON.stringify( [] ) );
+		localStorage.setItem(
+			selectedSiteId,
+			JSON.stringify( {
+				messages: [],
+				chatId: null,
+			} )
```
Comment (sejas) [inline src/hooks/use-assistant.ts:53]: Nice, I prefer to have the messages in a separate object.

**Comment by kozer** [comment]
Closing in favor of https://github.com/Automattic/studio/pull/230

**Pull request closed without merging**